src/settings.rs

```diff
@@ -102,6 +107,8 @@ static DEFAULT_EXCLUDE: Lazy<Vec<FilePattern>> = Lazy::new(|| {
     ]
 });
 
+static DEFAULT_DUMMY_VARIABLE_RGX: Lazy<Regex> = Lazy::new(|| Regex::new("^_$").unwrap());
```

Comment:
Should we make this `_+$|(_[a-zA-Z0-9_]*[a-zA-Z0-9]+?$)|dummy|^ignored_|^unused_` to match Pylint?

Reply:
(Will make this change then merge.)

Reply:
Yes, that sounds good to me :)

Reply:
(A went for something in-between -- I think `dummy` and `ignored` and `unused` are a bit odd as defaults.)

Reply:
Makes sense!